ThreeBotPackages/threefold/websites/package.py

```diff
@@ -0,0 +1,33 @@
+from Jumpscale import j
+
+
+class Package(j.baseclasses.threebot_package):
+    def start(self):
+        """
+        cl = j.clients.gedis.get("packagemanager", package_name="zerobot.packagemanager")
+        cl.actors.package_manager.package_add(path="/sandbox/code/github/threefoldtech/jumpscaleX_threebot/ThreeBotPackages/threefold/websites/")
+        """
+        repos = [
+            {"name": "www_3bot_org", "giturl": "https://github.com/threefoldtech/www_3bot_org.git", "branch": "3bot"},
+            {
+                "name": "www_threefold_love",
+                "giturl": "https://github.com/threefold-love/www_threefold_love",
+                "branch": "3bot",
+            },
+            {
+                "name": "www_threefold_tech",
+                "giturl": "https://github.com/threefoldtech/www_threefold.tech",
+                "branch": "3bot",
+            },
+            {"name": "www_incubaid", "giturl": "https://github.com/Incubaid/www_incubaid", "branch": "3bot"},
+            # {
+            #     "name": "www_threefold_io",
+            #     "giturl": "https://github.com/threefoldfoundation/www_threefold.io_new/",
```

Comment:
Here `giturl` needs to contain the full path to the package, not the repository root directory, `package.toml` is located under a directory named `package`, see https://github.com/threefoldfoundation/www_threefold.io_new/tree/3bot/package

Reply:
@xmonader Also suggested having all package related files inside the root of any website repository (like e.g. `package.toml`, `package.py`...).